configs/default/DIAT-MAMBAF722.config

```diff
@@ -114,5 +116,7 @@ set serialrx_provider = SBUS
 set blackbox_device = SPIFLASH
 set dshot_burst = ON
 set motor_pwm_protocol = DSHOT600
```

Comment:
To my knowledge this is sold as a dedicated flight controller without on-board ESCs - can you please remove this line to make it compliant with https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#13-donts?

Reply:
Waiting to hear back from Diatone.  ESCs not built into board, but pretty sure 90%+ of sales are complete rtf units with their own escs that have DHOT600 on default.  I believe that is why they want that in the config.  It’s been in the MAMBAF722 config since inception, so didn’t think it would be a problem keeping it in.  Your completely understand why it should not be there.   If they do not have a reason that is satisfactory to you then we will of course take it out.  Will comment when I hear from them.

Reply:
@mikeller Heard back from Diatone.  Exactly like I thought above.   They would like to keep DHOT600 in the config file, but understand it does not follow the rules because the board does not have build in ESCs supporting DSHOT.   There is no other argument to keep it in except the justification above, so if it must be removed then just say so and it's gone.  Code is waiting along with the name changes in the other pull request to push to you.  Thanks again and sorry for the issues.